tests/bootstrap.js

```diff
@@ -65,6 +70,24 @@ const waitForPg = async (url) => {
   }});
 };
 
+const isDBReady = (url = '') => {
```

Comment:
Don't default unless it actually provides value. In this case you've replaced one error with another, and it'll even keep retrying until the tests timeout (I guess?).

Much better to fail fast here. Add a check that runs in the `before` just before the `waitForCouch` call that throws if either of the required env vars are missing, and then you can assume they're set everywhere without default params.

Reply:
Nice! Done
<img width="365" alt="Screen Shot 2021-03-23 at 2 38 38 pm" src="https://user-images.githubusercontent.com/66472237/112110487-8a5c2b80-8be5-11eb-968d-52e8748a7f5b.png">